apis/firebase/rtdb/query context builder/e45854b1-aa4f-4bc0-bad0-04d0657de11d.json

```diff
@@ -5,20 +5,20 @@
       "3629005f-d095-4e3c-8d26-9987239d5d69": {
         "description": "Receives individual fields for syncing.",
         "paramId": "fields",
-        "type": "any",
+        "type": "string",
         "name": "ID token"
       },
       "f8ad245c-763e-47e8-9df2-ef1b439b7abd": {
         "description": "Receives individual fields for syncing.",
         "paramId": "fields",
-        "type": "any",
+        "type": "string",
         "name": "DB URL"
       }
     },
     "outputs": {
       "367a9897-be40-4dcf-854b-d565e32acfbf": {
         "description": "Sends synchronized inputs as a dictionary, indexed by field.",
-        "type": "{typeof(fields):any}",
+        "type": "{\n  \"ID token\": typeof `ID token`,\n  \"DB URL\" : typeof `DB URL`\n}",
```

Comment:
for me it a bit disturbing to use backtick (`) identify string in double quote string in a json. could we use single quote (') for this?

Reply:
Backtick is used to identify a reference to a port or node name.
This is the required syntax by type-descriptor.
When you refer to the type of port B when specifying the type port A, you use the expression:
`typeof ``B```